h/api/groups/models.py

```diff
@@ -65,3 +83,8 @@ def get_by_id(cls, id_):
               sa.ForeignKey('group.id'),
               nullable=False)
 )
+
+
+def includeme(config):
+    assert config.registry.settings.get("h.hashids.salt"), (
```

Comment:
I've changed this but it's confusing me, if I delete the h.hashids.salt line from development.ini then when it gets to this includeme() there _is_ a h.hashids.salt in the settings! And it's not coming from any environment variable. Where does this come from??